pkg/kubectl/resource_printer_test.go

```diff
@@ -276,12 +284,17 @@ func TestUnknownTypePrinting(t *testing.T) {
 
 func TestTemplateEmitsVersionedObjects(t *testing.T) {
 	// kind is always blank in memory and set on the wire
-	printer, err := NewTemplatePrinter([]byte(`{{.kind}}`), testapi.Version(), api.Scheme)
+	printer, err := NewTemplatePrinter([]byte(`{{.kind}}`))
```

Comment:
I deliberately made printers take the version to prevent the case where people wrote unversioned stuff out, this test was supposed to test that.